tasks/new_e2e_tests.py

```diff
@@ -182,6 +183,27 @@ def clean(ctx, locks=True, stacks=False, output=False):
         _clean_output()
 
 
+@task
+def deps(ctx, verbose=False):
+    """
+    Setup Go dependencies
+    """
+    max_retry = 3
+    print("downloading dependencies")
+    with timed("go mod download"):
+        verbosity = ' -x' if verbose else ''
+
+        with ctx.cd("test/new-e2e"):
+            for retry in range(max_retry):
+                result = ctx.run(f"go mod download{verbosity}")
+                if result.exited is None or result.exited > 0:
+                    wait = 10 ** (retry + 1)
+                    print(f"[{(retry + 1) / max_retry}] Failed downloading, retrying in {wait} seconds")
```

Comment:
```suggestion
                    print(f"[{retry + 1} / {max_retry}] Failed downloading, retrying in {wait} seconds")
```
Should this be like this ?

Reply:
Thanks! I'll fix the others in a follow up PR because I have a small refacto incoming since this is a copy/paste